egs/librispeech/ASR/pruned_transducer_stateless2/conformer.py

```diff
@@ -1098,7 +1098,7 @@ def multi_head_attention_forward(
             head_dim * num_heads == embed_dim
         ), "embed_dim must be divisible by num_heads"
 
-        scaling = float(head_dim) ** -0.5
+        scaling = torch.ones((1), dtype=torch.float32) * float(head_dim) ** -0.5
```

Comment:
Do we need the `device` argument here? I suspect it would raise an error when training.

Reply:
I tried to export with `device=cuda`, and it raises device mismatch errors, I am adding device argument.